projects/demo/src/modules/components/breadcrumbs/examples/2/index.ts

```diff
@@ -41,8 +41,8 @@ export default class Example {
             link: 'https://github.com/topics/angular',
         },
         {
-            caption: 'Tinkoff',
-            link: 'https://github.com/tinkoff',
+            caption: 'TramvaiJS',
```

Comment:
nit: `Angular > tramvai ? > Taiga > Components` 
It seems that the Tramvai doesn't fit the logic
maybe it's better to find other word, or just drop this